vendor.conf

```diff
@@ -33,7 +33,7 @@ github.com/opentracing/opentracing-go               1361b9cd60be79c4c3a7fa9841b3
 github.com/google/shlex                             6f45313302b9c56850fc17f99e40caebce98c716
 github.com/opentracing-contrib/go-stdlib            b1a47cfbdd7543e70e9ef3e73d0802ad306cc1cc
 github.com/mitchellh/hashstructure                  2bca23e0e452137f789efbc8610126fd8b94f73b
-github.com/gofrs/flock                              7f43ea2e6a643ad441fc12d0ecc0d3388b300c53 # v0.7.0
+github.com/gofrs/flock                              392e7fae8f1b0bdbd67dad7237d23f618feb6dbb # v0.7.1
```

Comment:
Haven't opened a PR in buildkit for this yet; let me know if you want me to do that @tonistiigi